code/game/gamemodes/game_mode.dm

```diff
@@ -110,9 +110,9 @@
 
 	var/list/living_crew = list()
 
-	for(var/mob/mob in player_list)
-		if(mob in living_mob_list && mob.ckey in joined_player_list)
-			living_crew += mob
+	for(var/mob/Player in mob_list)
+		if(Player.mind && Player.stat != DEAD && !isnewplayer(Player) &&!isbrain(Player))
+			living_crew++

```

Comment:
No idea what _list = list + 1_ is doing in Byond.
Just to make sure it's what was intended to be written.